tests/containers/single_container_webui.pm

```diff
@@ -9,9 +9,10 @@ sub run {
 
   assert_script_run("openssl req -newkey rsa:4096 -x509 -sha256 -days 365 -nodes -subj '/CN=www.mydom.com/O=My Company Name LTD./C=DE' -out server.crt -keyout server.key");
 
-  assert_script_run("docker run --rm -d --network testing $volumes $certificates -p 80:80 --name openqa_webui openqa_webui");
+  assert_script_run("docker run -d --network testing $volumes $certificates -p 80:80 --name openqa_webui openqa_webui");
   wait_for_container_log("openqa_webui", "Web application available at", "docker");
-
+  assert_script_run("docker rm -f openqa_webui");
+  
   assert_script_run("curl http://localhost");
   assert_script_run("docker rm -f openqa_webui");
```

Comment:
This is now done twice. I'm not quite sure what the intention is but this looks quite odd.

Reply:
Yes, was a mistake